lesson17/index.html

```diff
@@ -5,15 +5,12 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>lesson 16</title>
+  <title>lesson 17</title>
```

Comment:
IMO
It is recommended to use the defer attribute.
[Reference](https://penpen-dev.com/blog/script-async-defer/)
```suggestion
  <script src="./index.js" defer></script>
  <title>lesson 17</title>
```

Reply:
[37e1507](https://github.com/hippopo-moo/js-handson-work/pull/19/commits/37e1507ef38aec4fc00b93ceb96256c0257175e7)

Sure looks good to me, so I fixed it!
(This was a good opportunity to look into defer)